frontend/app/controllers/registration.js

```diff
@@ -0,0 +1,20 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+
+export default Controller.extend({
+  notify: service(),
+
+  actions: {
+    createUser() {
+      if (this.get("model.password") === this.get("confirmPassword")) {
+        this.get("model")
+          .save()
+          .then(() => {
+            this.transitionToRoute("login");
+          });
+      } else {
+        this.get("notify").error("Passwords are not equal");
```

Comment:
`equal` is a word used in maths, not for a user message.. Use something like "Passwords do not match"